src/main/java/com/commercetools/sync/producttypes/utils/AttributeDefinitionsUpdateActionUtils.java

```diff
@@ -101,8 +95,8 @@
 
             updateActions.addAll(
                 buildAddAttributeDefinitionUpdateActions(
-                    newAttributeDefinitionsDrafts,
-                    oldAttributesDefinitionsNameMap
+                    oldAttributeDefinitions,
```

Comment:
**Important**: We should definitely filter the input lists (both) from any `null` values (best at the entry method: `buildAttributeDefinitionsUpdateActions`) because it is possible and could cause NPEs at several points.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/316/commits/b74420a1e3a065c11f20b5337b6d5f9f0707e05a